custom_components/rivian/__init__.py

```diff
@@ -158,54 +151,48 @@ async def _update_api_data(self):
 
         sensors.append("gnssLocation")
         try:
-            auth = await self._api.authenticate_graphql(
-                self._entry.data.get(CONF_USERNAME), self._entry.data.get(CONF_PASSWORD)
-            )
+            if self._login_attempts >= 4:
```

Comment:
If we get to 4 login attempts.... how do we get out of the aborted situation?

Reply:
I want to add a timed reset, but really put this here to prevent overloading the servers on error. Since any error we previously were just authenticating again I did have a time where I sent by mistake 1000s of authentication requests 😨.

Right now it gets reset by going through configuration again.